src/renderer/components/reader/ReaderOptions.tsx

```diff
@@ -197,90 +197,131 @@ export class ReaderOptions extends React.Component<Props> {
-                        onClick={(e) => this.props.handleSettingChange(e, "paged", "true")}
-                    >
-                        <SVG svg={PagineIcon} title={__("reader.settings.paginated")}/>
-                        {__("reader.settings.paginated")}
-                    </button>
-                </div>
-                </div>
-                <div className={styles.line_tab_content}>
+            <section className={styles.line_tab_content}>
+            <div className={styles.subheading}>{__("reader.settings.disposition.title")}</div>
+                <ul className={styles.center_in_tab}>
+                    <li>
+                        <label htmlFor={styles.scroll_option}
+                        className={this.getButtonClassName("paged", false)}>
+                            <SVG svg={DefileIcon} title={__("reader.settings.scrolled")} />
+                            {__("reader.settings.scrolled")}
+                        </label>
+                        <input
+                        id={styles.scroll_option}
+                        type="radio"
+                        name="disposition"
+                        onChange={(e) => this.props.handleSettingChange(e, "paged", "false")}
+                        />
+                    </li>
+                    <li>
+                        <label htmlFor={styles.page_option}
+                        className={this.getButtonClassName("paged", true)}>
+                            <SVG svg={PagineIcon} title={__("reader.settings.paginated")}/>
+                            {__("reader.settings.paginated")}
+                        </label>
+                        <input
+                        id={styles.page_option}
+                        type="radio"
+                        name="disposition"
+                        onChange={(e) => this.props.handleSettingChange(e, "paged", "true")}
+                        defaultChecked={true}
+                        />
+                    </li>
+                </ul>
+            </section>
+            <section className={styles.line_tab_content}>
                 <div className={styles.subheading}>{__("reader.settings.justification")}</div>
-                <div className={styles.center_in_tab}>
-                    <button
-                        id={styles.option_gauche} role="link"
-                        className={this.getButtonClassName("align", "left")}
+                <ul className={styles.center_in_tab}>
+                    <li>
+                        <label htmlFor={"radio-" + styles.option_gauche}
+                        className={this.getButtonClassName("align", "left")}>
+                            <SVG svg={LeftIcon} title={__("reader.settings.left")}/>
+                            {__("reader.settings.left")}
+                        </label>
+                        <input
+                        id={"radio-" + styles.option_gauche}
+                        name="alignement"
```

Comment:
English is `alignment` :) (no `e`)

Reply:
PS: I am aware that there are many other French words (such as `option_gauche` and `option_colonne`), and it must be very tempting to fix this now in these small PRs ... but hopefully this will be addressed in a future refactoring :)